spaceward/src/features/actions/util.ts

```
@@ -183,45 +179,47 @@ export const handleEthRaw = async ({
 					// fixme
 				})
 				.then(() => true);
 		case "snap":
+			if (!action.snap) {
+				throw new Error("snap params not set");
+			}
+
 			const keyringSnapClient = new KeyringSnapRpcClient(
 				env.snapOrigin,
 				window.ethereum,
 			);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)